src/main/java/com/bobocode/svydovets/annotation/annotations/AutoSvydovets.java

```diff
@@ -0,0 +1,12 @@
+package com.bobocode.svydovets.annotation.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
```

Comment:
As we discussed, we decided to support field, constructor, and setter-based injections.
So, the target should be `{ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.FIELD}`